pkg/varlog/options.go

```diff
@@ -182,6 +185,12 @@ func withoutSelectLogStream() AppendOption {
 	})
 }
 
+func WithAllowLogStreams(logStreams map[types.LogStreamID]struct{}) AppendOption {
```

Comment:
WithAllowedLogStreams seems to be a more natural expression.
```go
func WithAllowedLogStreams(logStreams map[types.LogStreamID]struct{}) AppendOption {
```

Reply:
okay